articles/multifactor-authentication/google-auth/admin-guide.md

```diff
@@ -4,6 +4,10 @@ description:  Using Google Authenticator with Auth0 for administrators
 
 # Google Authenticator for Administrators
 
+::: warning
+The Google Authenticator provider is deprecated. You should use [Guardian](/multifactor-authentication/guardian) instead, which supports TOTP (*including Google Authenticator*), SMS, and push authentication using the [Guardian App](/multifactor-authentication/guardian/user-guide). We will send a deprecation notification and  an upgrade path for existing users in the future.
+:::
+
 ::: note
 This page refers to using Google Authenticator instead of Auth0 Guardian. Google Authenticator can also be used with Guardian (your users choose which to use) when **Push Notifications** is enabled. [Click here for more information on Guardian.](/multifactor-authentication/guardian)
 :::
```

Comment:
Maybe it's me, but these two panels don't make much sense when you read one after the other. I'd remove the second one, the first says the same thing, plus some more. Wdyt?

Reply:
Also, in this sentence `You should use [Guardian](/multifactor-authentication/guardian) instead, which supports TOTP (*including Google Authenticator*), SMS, and push authentication using the [Guardian App](/multifactor-authentication/guardian/user-guide).`

the ` using the [Guardian App](/multifactor-authentication/guardian/user-guide)` bit at the end looks redundant. The sentence already starts with `You should use Guardian`.

Reply:
Both comments apply to all 3 docs. Also, when we re-use stuff we use [document fragments](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#reusing-content)

Reply:
Feedback applied! Thanks :).